Traceback (most recent call last):
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\urllib3\util\connection.py", line 73, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\SideProject_IT\car_price estimated\cargurus_spider.py", line 58, in <module>
    driver.quit()
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\selenium\webdriver\chromium\webdriver.py", line 221, in quit
    super().quit()
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 616, in quit
    self.execute(Command.QUIT)
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 455, in execute
    response = cast(RemoteConnection, self.command_executor).execute(driver_command, params)
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 405, in execute
    return self._request(command_info[0], url, body=data)
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\selenium\webdriver\remote\remote_connection.py", line 429, in _request
    response = self._conn.request(method, url, body=body, headers=headers, timeout=self._client_config.timeout)
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\urllib3\_request_methods.py", line 135, in request
    return self.request_encode_url(
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\urllib3\_request_methods.py", line 182, in request_encode_url
    return self.urlopen(method, url, **extra_kw)
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\urllib3\poolmanager.py", line 459, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\urllib3\connectionpool.py", line 787, in urlopen
    response = self._make_request(
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\urllib3\connectionpool.py", line 493, in _make_request
    conn.request(
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\urllib3\connection.py", line 494, in request
    self.endheaders()
  File "C:\Users\creat\AppData\Local\Programs\Python\Python310\lib\http\client.py", line 1278, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Users\creat\AppData\Local\Programs\Python\Python310\lib\http\client.py", line 1038, in _send_output
    self.send(msg)
  File "C:\Users\creat\AppData\Local\Programs\Python\Python310\lib\http\client.py", line 976, in send
    self.connect()
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\urllib3\connection.py", line 325, in connect
    self.sock = self._new_conn()
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\urllib3\connection.py", line 198, in _new_conn
    sock = connection.create_connection(
  File "D:\SideProject_IT\car_price estimated\.venv\lib\site-packages\urllib3\util\connection.py", line 73, in create_connection
    sock.connect(sa)
KeyboardInterrupt
